tests/ui/rfcs/rfc-3348-c-string-literals/auxiliary/wrong_parsing.rs

```diff
@@ -0,0 +1,18 @@
+// force-host
+// no-prefer-dynamic
+#![crate_type = "proc-macro"]
+
+extern crate proc_macro;
+
+use proc_macro::{TokenStream, TokenTree};
+
+#[proc_macro]
+pub fn repro(input: TokenStream) -> TokenStream {
+    for token in input {
+        if let TokenTree::Literal(literal) = token {
+            assert!(format!("{}", literal).contains(&"b\""), "panic on: `{}`", literal);
```

Comment:
This should be checking that a cstring starts with `c\"`?

Reply:
Good catch, thanks I commit the wrong version of the test